src/types/routes.ts

```diff
@@ -0,0 +1,3 @@
+import { Route, RouteExtended } from '@lifi/sdk';
+
+export type RouteVariant = Route | RouteExtended;
```

Comment:
RouteExtended is already inheriting from Route, this new type might not give any benefits (+ naming does not match it's type)